api/src/main/java/io/strimzi/api/kafka/model/MaintenanceSpec.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MaintenanceSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String schedule;
+
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("The schedule for the maintenance tasks. It's defined by a cron expression.")
```

Comment:
Is there some reliable definition of the Cron format where we could link?

Reply:
I'll search for a feasible reference.

Reply:
I didn't find a reliable website for that but I see that "cron expression" or "cron format" is something widely used so people can search for it.